### PR TITLE
MODDATAIMP-12: Fix result block behavior.

### PR DESCRIPTION
**Pull request by OleksandrAntonenko1**

### CODE REVIEW

File: src/settings/index.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import Settings from '@folio/stripes-components/lib/Settings';
+import Settings from '@folio/stripes-smart-components/lib/Settings';
```
Comment (VictorSoroka1) [inline src/settings/index.js]: Please change the import to be `{ Settigns } from ...`.

File: src/components/ResultPanel/ResultPanel.css [hunk 2]
```diff
@@ -39,6 +40,8 @@
 
 .accordionHeader {
   width: 100%;
+  flex-wrap: wrap;
```
Comment (VictorSoroka1) [inline src/components/ResultPanel/ResultPanel.css]: Let's have it above the width.

File: src/components/ResultPanel/ResultPanel.css [hunk 3]
```diff
@@ -2,6 +2,7 @@
 
 .recordType {
   display: inline-block;
+  flex-shrink: 0;
```
Comment (VictorSoroka1) [inline src/components/ResultPanel/ResultPanel.css]: Let's have above the display. BTW, do we need display there as it seems to be flex item?

File: src/components/ResultPanel/ResultPanel.css (hunk 2 above)
Comment (OleksandrAntonenko1) [reply src/components/ResultPanel/ResultPanel.css]: Done.

File: src/components/ResultPanel/ResultPanel.css (hunk 3 above)
Comment (OleksandrAntonenko1) [reply src/components/ResultPanel/ResultPanel.css]: Done.

**Pull request merged**